packages/components/stories/DateTimePicker.js

```diff
@@ -2,11 +2,24 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { IconsProvider } from '../src/index';
 
-import DateTimePicker from '../src/DateTimePickers';
+import DateTimePicker, { InputDateTimePicker } from '../src/DateTimePickers';
 
 
 storiesOf('DateTimePicker', module)
-	.add('Full DateTimePicker', () => (
+	.add('InputDateTimePicker', () => (
+		<div>
+			<h1>InputDateTimePicker</h1>
+			<IconsProvider />
+
+			<div>
+				<InputDateTimePicker
+					selectedDateTime={new Date(2018, 4, 13, 12, 30)}
+					onChange={console.log}
```

Comment:
use storybook actions to make it appear in the action logger panel

Reply:
great, I didn't know those things :)

Reply:
✔